code/routers/user/auth.js

```diff
@@ -102,16 +105,26 @@ router.post('/signup', function(req, res) {
       newUser.email = email;
       newUser.setPassword(password);
 
-      newUser.save(function(err, user) {
-        if(err) {
-          return res.status(authCode.signup.failedToAdd.status).send({
-            message: authCode.signup.failedToAdd.message
-          });
-        } else {
-          return res.status(authCode.signup.success.status).send({
-            message: authCode.signup.success.message
-          });
-        }
+      locationQuery = {
```

Comment:
I this going to stay there for now?

Reply:
Thank you for pointing this out. Changing this to what it should be